apps/application-system/api/src/app/modules/application/dto/populateExternalData.dto.ts

```diff
@@ -10,6 +10,10 @@ class DataProviderDto {
   @IsString()
   @ApiProperty()
   type!: string
+
+  @ApiPropertyOptional()
+  @IsNumber()
+  order?: number
```

Comment:
Can we make this an optional configuration on the TemplateApiAction definition? Since templates can override it there for their needs. Better to keep this out of the API so users can't tamper with it.